test/parallel/test-http-methods.js

```diff
@@ -23,13 +23,46 @@
 require('../common');
 const assert = require('assert');
 const http = require('http');
-const util = require('util');
 
+// This test ensures all http methods from HTTP parser are exposed 
+// to http library
+
+var methods = [
+  'DELETE',
+  'GET',
+  'HEAD',
+  'POST',
+  'PUT',
+  'CONNECT',
+  'OPTIONS',
+  'TRACE',
+  'COPY',
+  'LOCK',
+  'MKCOL',
+  'MOVE',
+  'PROPFIND',
+  'PROPPATCH',
+  'SEARCH',
+  'UNLOCK',
+  'BIND',
+  'REBIND',
+  'UNBIND',
+  'ACL',
+  'REPORT',
+  'MKACTIVITY',
+  'CHECKOUT',
+  'MERGE',
+  'M-SEARCH',
+  'NOTIFY',
+  'SUBSCRIBE',
+  'UNSUBSCRIBE',
+  'PATCH',
+  'PURGE',
+  'MKCALENDAR',
+  'LINK',
+  'UNLINK'
+];
 assert(Array.isArray(http.METHODS));
 assert(http.METHODS.length > 0);
-assert(http.METHODS.includes('DELETE'));
-assert(http.METHODS.includes('GET'));
-assert(http.METHODS.includes('HEAD'));
-assert(http.METHODS.includes('POST'));
-assert(http.METHODS.includes('PUT'));
-assert.deepStrictEqual(util._extend([], http.METHODS), http.METHODS.sort());
+assert.equal(methods.length, http.METHODS.length);
+assert.deepStrictEqual(methods.sort(), http.METHODS.sort());
```

Comment:
Aside from that, can you please swap the order of arguments here? According to the function's signature, first argument should be an actual value and second one an expected value.